packages/react-apollo/mixins/apollo2/mixin.browser.js

```diff
@@ -6,6 +6,7 @@ const {
   },
 } = require('hops-mixin');
 
+// eslint-disable-next-line node/no-extraneous-require
```

Comment:
We need to disable this rule here, because we have removed the peer dependency to react-apollo.